docs/topics/mta-web-creating-migration-waves.adoc

```diff
@@ -4,36 +4,27 @@
 
 :_content-type: PROCEDURE
 [id="mta-web-creating-migration-waves_{context}"]
-= Creating migration waves
+= Creating a migration wave
 
-Creating a migration wave is a way to group applications that you want to migrate on a given schedule.
-
-In addition, a migration wave enables you to track each migration by exporting a list of the wave's applications to the Jira issue management system. This automatically creates a separate Jira issue for each application of the migration wave.
+By creating a migration wave, you can group applications that you want to migrate on a given schedule. You can also track each migration by exporting a list of the wave's applications to the Jira issue management system. This automatically creates a separate Jira issue for each application of the migration wave.
```

Comment:
Define what a migration wave is.

Reply:
@anarnold97, do you have the definition in some other doc, please? Would be very helpful.

Reply:
https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/7.0/html/user_interface_guide/working-with-applications-in-the-ui#mta-web-creating-migration-waves_user-interface-guide

```
 Creating a migration wave is a way to group applications that you want to migrate on a given schedule.

In addition, a migration wave enables you to track each migration by exporting a list of the wave’s applications to the Jira issue management system. This automatically creates a separate Jira issue for each application of the migration wave. 
```

Will this do?